app/grandchallenge/archives/forms.py

```diff
@@ -225,3 +232,37 @@ def save(self, *args, **kwargs):
             {"interface_pk": self.cleaned_data["interface"].pk}
         )
         return super().save(*args, **kwargs)
+
+
+class ArchiveItemFormMixin(MultipleCIVForm):
```

Comment:
Does this need to inherit from `MultipleCIVForm`? Now `ArchiveItemCreateForm` basically inherits `MultipleCIVForm` twice, once through `ArchiveItemCreateForm` and once directly. Have you tested that this works as expected?

Reply:
Def doesn't need it. Refactor oversight on removing it. The DisplaySet's homologue doesn't use it either.